build.xml

```diff
@@ -515,6 +606,7 @@
         <jvmarg value="-Dstorage-config=${test.conf}"/>
         <jvmarg value="-Dcassandra.reads.thresholds.coordinator.defensive_checks_enabled=true" /> <!-- enable defensive checks -->
         <jvmarg value="-javaagent:${build.dir.lib}/jars/jamm-${jamm.version}.jar" />
+        <jvmarg value="-Dnet.bytebuddy.experimental=true"/>
```

Comment:
none of the byte buddy -D should be needed as we are using version 1.14.17 which supports up to JDK23

Reply:
Check this comment:
https://github.com/datastax/cassandra/pull/1408#discussion_r1840883671
Issues with mockito-inline